VTEX - Logistics API.json

```diff
@@ -8085,43 +8096,83 @@
                     "content": {
                         "application/json": {
                             "schema": {
-                                "title": "Create/UpdatePolygonRequest",
+                                "type": "object",
                                 "required": [
                                     "name",
                                     "geoShape"
                                 ],
-                                "type": "object",
                                 "properties": {
                                     "name": {
-                                        "type": "string"
+                                        "type": "string",
+                                        "description": "Name of the [polygon](https://help.vtex.com/en/tutorial/registering-geolocation--tutorials_138) being created or updated.",
+                                        "example": "ipanema_leblon"
                                     },
-                                    "geoShape": {
-                                        "title": "GeoShape",
+                                    "geoShape": {                            
+                                        "type": "object",
+                                        "description": "Polygon geolocation coordinates.",
                                         "required": [
                                             "coordinates"
                                         ],
-                                        "type": "object",
                                         "properties": {
                                             "coordinates": {
                                                 "type": "array",
+                                                "description": "List of geolocation coordinates that make part of the polygon. The first and the last pair of coordinates must be the equal.",
                                                 "items": {
                                                     "type": "array",
+                                                    "description": "Polygon geolocation coordinates, first longitude then latitude.",
                                                     "items": {
-                                                        "type": "number"
+                                                        "type": "number",
+                                                        "description": "Longitude or latitude.",
+                                                        "example": -43.231716156005859
                                                     }
-                                                }
+                                                },
+                                                "example": [
+                                                    [
+                                                        [
```

Comment:
There is an extra `[]` in this sequence. Hence the spectral alert.

It should be 
"example": [[-43.231716156005859, -22.990713500405295],[...],[...]]

instead of
"example": [[[-43.231716156005859, -22.990713500405295],[...],[...]]]

Reply:
Ok, but then it means the schema is wrong. There should be one extra `items` layer.